vernac/declare.mli

```diff
@@ -384,14 +384,18 @@ val declare_entry
   -> proof_entry
   -> GlobRef.t
 
+(** Declaration of section variables and local definitions *)
+type variable_declaration =
+  | SectionLocalDef of { clearbody : bool; entry : proof_entry }
+  | SectionLocalAssum of { typ:Constr.types; impl:Glob_term.binding_kind ;
+                           univs:UState.named_universes_entry }
```

Comment:
something like 
```suggestion
  | SectionLocalAssum of {
      typ:Constr.types;
      impl:Glob_term.binding_kind ;
      univs:UState.named_universes_entry
      }
```
is nicer IMO (I didn't count the spaces as I wrote it in the github UI)

Reply:
Adopted (as well as for `SectionLocalDef` then).

PS: not relaunching full ci as it was already found green.